packageregistry/git.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"strings"
 
-	giturls "github.com/whilp/git-urls"
+	vcsurl "github.com/gitsight/go-vcsurl"
```

Comment:
Can we get rid of both `github.com/gitsight/go-vcsurl` and `github.com/whilp/git-urls`? They seem to be unmaintained and has very less usage.

Alternatively, we can fork and maintain `github.com/whilp/git-urls` under SafeDep GitHub org where we will fix vulnerabilities. But do we really need such a library? Looking at the code, they hardly do anything

https://github.com/whilp/git-urls/blob/master/urls.go